pathology/nuclick/nuclei_classification_training_notebook.ipynb

```diff
@@ -145,7 +145,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Configure Workspace Path\n",
+    "## Configure dataset path and visualize\n",
```

Comment:
Configure **the** dataset path and visualize *a sample image**

Reply:
Thanks, updated it in the latest commit.